README.md

```diff
@@ -68,6 +68,31 @@ metadata:
 type: Opaque
 ```
 
+Your Exoscale Key should have at least the following `operation` for your domain: `list-dns-domain-records`, `get-dns-domain-record`, `list-dns-domains`, `update-dns-domain-record`, `create-dns-domain-record`, `delete-dns-domain-record`
+
+Here is an example of the minimum policy required for the Exoscale Key:
+
+```json
+{
+  "default-service-strategy": "deny",
+  "services": {
+    "dns": {
+      "type": "rules",
+      "rules": [
+        {
+          "expression": "resources.dns_domain.unicode_name!=\"example.com\"",
+          "action": "deny"
+        },
+        {
+          "expression": "operationin['list-dns-domain-records','get-dns-domain-record','list-dns-domains','update-dns-domain-record','create-dns-domain-record','delete-dns-domain-record']",
```

Comment:
```suggestion
          "expression": "operation in['list-dns-domain-records','get-dns-domain-record','list-dns-domains','update-dns-domain-record','create-dns-domain-record','delete-dns-domain-record']",
```

not your fault, this is a bug where some whitespace is removed when you export an IAM policy from the Portal. We're working on i.

Reply:
Is there a reason you didn't include the `get-operation` in this list? Does the webhook work without it?
(`get-operation` is listed on line 16)

Reply:
It looks like you added `update-dns-domain-record` in addition to what we recommend on line 16. Do we really need it?

Reply:
After checking the source code I'm certain, that we don't need the `update-dns-domain-record` operation. However we do need the `get-operation`.